h/features.py

```diff
@@ -108,8 +142,8 @@ def features_status(request):
 
 
 def includeme(config):
+    # Remove old feature flags from the database on startup
+    config.action(None, remove_old_flags, order=90)
```

Comment:
It's not supposed to be an absolute ordering, and pyramid modules are supposed to be (mostly) independent of one another, so no. I'm not sure the value of such a thing would outweigh the indirection costs at the moment.